lib/lookup.json

```diff
@@ -300,7 +300,6 @@
   },
   "node-sass": {
     "replace": true,
-    "master": true,
```

Comment:
you will need to include `"prefix": "v"` so our algorithm can find the tarball on github